include/mbedtls/aria.h

```diff
@@ -299,17 +323,24 @@ int mbedtls_aria_crypt_cfb128( mbedtls_aria_context *ctx,
  *             securely discarded as soon as it's no longer needed.
  *
  * \param ctx              The ARIA context to use for encryption or decryption.
- * \param length           The length of the input data.
+ *                         This must be initialized and bound to a key.
+ * \param length           The length of the input data \p input in Bytes.
  * \param nc_off           The offset in the current \p stream_block, for
  *                         resuming within the current cipher stream. The
  *                         offset pointer should be 0 at the start of a stream.
- * \param nonce_counter    The 128-bit nonce and counter.
- * \param stream_block     The saved stream block for resuming. This is
- *                         overwritten by the function.
+ *                         This must not be larger than 15.
```

Comment:
We shouldn't just give a number, but number and unit. Therefore, shouldn't it be:
> This must not be larger than 15 bytes.

Reply:
In the end, you pass a unit-less number, so I would have preferred not to add this here, but I'm also ok changing it.